main/test/ca/uwaterloo/flix/language/phase/TestWeeder.scala

```diff
@@ -1197,36 +1273,42 @@ class TestWeeder extends AnyFunSuite with TestUtils {
     expectError[WeederError.MalformedChar](result)
   }
 
+  // TODO: Still ignore or remove?
```

Comment:
I don't know why these are ignored. If the float is too big we should definitely give an error!

Reply:
I would say in the lexer.